### PR TITLE
Add collector to get table stats grouped by schema

### PR DESCRIPTION
**Pull request by maeserichar**
Adding a new collector to get the table stats info (if available) grouped by schema.

@SuperQ

### CODE REVIEW

**Comment by dannyk81** [comment]
ping @SuperQ 😄

File: collector/info_schema_schemastats.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+// Scrape collects data from database connection and sends it over channel as prometheus metric.
+func (ScrapeSchemaStat) Scrape(ctx context.Context, db *sql.DB, ch chan<- prometheus.Metric) error {
+	var varName, varVal string
+
+	err := db.QueryRowContext(ctx, userstatCheckQuery).Scan(&varName, &varVal)
+	if err != nil {
+		log.Debugln("Detailed schema stats are not available.")
+		return nil
+	}
+	if varVal == "OFF" {
```
Comment (SuperQ) [inline collector/info_schema_schemastats.go]: I see this in other collectors, perhaps we should refactor this out into a helper func. But fine for this review.

**Comment by SuperQ** [comment]
Please add a `[FEATURE]` line to the CHANGELOG.md and

**Comment by maeserichar** [comment]
Looks like the failing check does not have nothing to do with my changes. @SuperQ Can you relaunch it? Thanks!

**Comment by SuperQ** [comment]
Hrm, it's still failing. Can you make sure your branch is rebased against the latest master? We made some fixes to the build system.

**Comment by maeserichar** [comment]
Yes, of course. Thanks!

**Comment by maeserichar** [comment]
ping @SuperQ

**Pull request merged**